src/Akeneo/Connectivity/Connection/back/tests/Integration/Persistence/Dbal/Query/DbalGetAConnectionWebhookQueryIntegration.php

```diff
@@ -0,0 +1,96 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\back\tests\Integration\Persistence\Dbal\Query;
+
+use Akeneo\Connectivity\Connection\back\tests\Integration\Fixtures\ConnectionLoader;
+use Akeneo\Connectivity\Connection\Domain\Settings\Model\Read\ConnectionWithCredentials;
+use Akeneo\Connectivity\Connection\Domain\Settings\Model\ValueObject\FlowType;
+use Akeneo\Connectivity\Connection\Domain\Webhook\Model\Read\ConnectionWebhook;
+use Akeneo\Connectivity\Connection\Domain\Webhook\Persistence\Query\GetAConnectionWebhookQuery;
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use Doctrine\DBAL\Connection as DbalConnection;
+use PHPUnit\Framework\Assert;
+
+class DbalGetAConnectionWebhookQueryIntegration extends TestCase
+{
+    /** @var ConnectionLoader */
+    private $connectionLoader;
+
+    /** @var GetAConnectionWebhookQuery */
+    private $getAConnectionWebhookQuery;
+
+    /** @var DbalConnection */
+    private $dbalConnection;
+
+    public function test_it_gets_an_enabled_connection_webhook_in_terms_of_a_given_code(): void
```

Comment:
I'm not sure of the "in terms of" sense here
```suggestion
    public function test_it_gets_an_enabled_connection_webhook_for_a_given_code(): void
```

Reply:
in terms of = en fonction de